weed/storage/volume.go

```diff
@@ -195,6 +195,10 @@ func (v *Volume) Close() {
 	v.dataFileAccessLock.Lock()
 	defer v.dataFileAccessLock.Unlock()
 	if v.nm != nil {
+		for v.isCompacting {
```

Comment:
With this code, when the volume is committing, it can not close current volume.

Reply:
This is the point, since closing the volumes causes when the volume server is stopped. But it seems it's worth taking it to the stop level above.